src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { BsChatText, BsHandThumbsUp } from "react-icons/bs";
+
+const ArticleCard = ({
+  articleData: {
+    title,
+    topic,
+    author,
+    created_at,
+    article_img_url,
+    comment_count,
+    votes,
+  },
+}) => {
+  return (
+    <section className="articleCard">
+      <header>
+        <p>{author}</p>
+        <p>{created_at.slice(0, 10)}</p>
+      </header>
+      <main>
+        <p>{title}</p>
```

Comment:
It seems strange to have the article in a 'p' tag. Can you think of a tag that would be better suited for a title?